src/icons/Fem.js

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import PropTypes from 'prop-types'
+function SvgFem({ size, color, ...style }) {
```

Comment:
Please and line breaks to this file as well.

Reply:
Done. Also import React from 'react'